### ISSUE TITLE
CSS background-image is not rendering the image

### ISSUE THREAD

**Issue by rkoshy** [litehtml/litehtml#89]
I have the following style:
```
.state- {
    background-color: #f89f4f;
    /*background-image: url(icon-outgoing.svg);*/
    background-image: url('data:image/svg+xml;utf8.......');
}
```
The HTML:
```
<!DOCTYPE html>
<html>
<head>
    <link rel="stylesheet" type="text/css" href="styles.css">
    <meta charset="utf-8">
</head>
<body>
    <span class="state state-"></span>
</body>
</html>
```
The background block renders, but there is no attempt to get the information about the SVG (get_image_size).  Furthermore, no draw methods are called to render that SVG.

If you could guide me on where I should be looking, I am willing to troubleshoot and attempt to fix.

I've combed the code and do not see anywhere obvious that would be responsible for processing this attribute.

**Comment by tordex** [comment]
There was a bug with CSS parsing. I've fixed it in commit d79e800.

**Comment by rkoshy** [comment]
Awesome!  Thank you for the quick fix.
